instance/models/instance.py

```diff
@@ -157,6 +157,11 @@ class Instance(ValidateModelMixin, models.Model):
         blank=True,
         help_text='Custom tags associated with the instance.',
     )
+    accepted_privacy_policy = models.DateTimeField(
```

Comment:
@jamestait, this field should be in the registration model because that imho is the correct place. What do you think?

Reply:
Yes, that does seem like a more natural place for it. I'll move it.

Reply:
Done in the latest commit. :)